frontend/src/templates/Field/Checkbox/CheckboxField.tsx

```diff
@@ -77,6 +78,11 @@ export const CheckboxField = ({
     [checkboxInputName, getValues],
   )
 
+  useEffect(
+    () => resetField(checkboxInputName, { defaultValue: '' }),
+    [checkboxInputName, resetField],
+  )
```

Comment:
could we achieve the same outcome without using useEffects? This seems overkill. Could we try assigning a `defaultValue` prop in the components instead?

```tsx 
<Checkbox
  colorScheme={fieldColorScheme}
  key={idx}
  value={o}
  defaultValue="" // <----
  {...register(checkboxInputName, validationRules)}
>
```

```tsx
<FieldContainer schema={schema} errorKey={radioInputName}>
  <Controller
    name={radioInputName}
    rules={validationRules}
    defaultValue="" // <------
    render={({ field: { ref, onChange, value, ...rest } }) => (
      <Radio.RadioGroup
        colorScheme={fieldColorScheme}
```

for safety, should we add a default value to all field templates?

Reply:
yeah, adding the default value would definitely work haha

As for adding a default value to all fields, in some cases it is possible and some not. I'll add defaults to all possible cases.